src/OSPSuite.Presentation/Extensions/CurveChartExtensions.cs

```diff
@@ -28,10 +28,14 @@ public static Color SelectNewColor(this CurveChart chart)
          return colors[newIndex];
       }
 
-      public static void UpdateCurveColorAndStyle(this CurveChart chart, Curve curve, DataColumn dataColumn, IReadOnlyCollection<DataColumn> dataColumns)
+      public static void UpdateCurveColorAndStyle(this CurveChart chart, Curve curve, DataColumn dataColumn, IReadOnlyCollection<DataColumn> dataColumns, bool isLinkedDataToSimulation = false)
```

Comment:
I don't understand what this flags mean. CAn you eplain "isLinkedDataToSimulation"?